docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,576 +1,645 @@
+|---------------------|----------------------------|--------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| `Shape`             | `PrimitiveVisualShapeType` | all                                  | The shape to render. The default value is `Sphere`. For more information, see the [table](#table_primit_possible_values). |
+| `MixColor`          | `Color`                    | all                                  | The color of the shape. The default value is `[0.5f, 0.5f, 0.5f, 1.0f]`.                                           |
+| `Slices`            | `int` in a range 1-255     | all                                  | The number of slices going around the shape. The default value is `128`.                                           |
+| `Stacks`            | `int`                      | all                                  | The number of layers going down the shape. The default value is `128`.                                             |
+| `ScaleTopRadius`    | `float` >= 0               | conical frustrum                     | The scale of the top circle radius of a conical frustrum. The default value is `1.0`.                              |
+| `ScaleBottomRadius` | `float` >= 0               | conical frustrum<br>cone             | The scale of the bottom circle radius. The default value is `1.5`.                                                 |
+| `ScaleHeight`       | `float` >= 0               | conical frustrum<br>cone<br>cylinder | The height scale of the conic. The default value is ``3.0`.                                                        |
+| `ScaleRadius`       | `float` >= 0               | cylinder                             | The radius scale of the cylinder. The default value is `1.0`.                                                      |
+| `ScaleDimensions`   | `Vector3`                  | cube<br>octahedron<br>bevelled cube  | The dimensions of the cuboid. Scales in the same way as for a 9-patch image. The default value is `[1, 1, 1]`.     |
+| `BevelPercentage`   | `float` in a range 0-1     | bevelled cube                        | Specifies how bevelled the cuboid should be, based on the smallest dimension. It affects the ratio of the outer face widths to the width of the overall cube. The default value is `0.0`, which specifies no bevel. |
+| `BevelSmoothness`   | `float` in a range 0-1     | bevelled cube                        | Specifies how smooth the bevelled edges should be. The default value is `0.0`, which specifies sharp edges.        |
+| `LightPosition`     | `Vector3`                  | all                                  | The absolute position of the light source. The point from which the object is illuminated. All zeros indicate the top-left corner in the visual plane. |
 
-**Table: Mesh visual properties**
+There are seven `Shape` values, some of which are specializations as compared to another:
```

Comment:
I feel we need to rephrase this sentence to give it better meaning.

Reply:
added